library/core/src/intrinsics.rs

```diff
@@ -2654,6 +2653,17 @@ pub const unsafe fn const_allocate(_size: usize, _align: usize) -> *mut u8 {
 #[cfg_attr(not(bootstrap), rustc_intrinsic)]
 pub const unsafe fn const_deallocate(_ptr: *mut u8, _size: usize, _align: usize) {}
 
+/// `ptr` must point to a vtable.
+/// The intrinsic will return the size stored in that vtable.
+#[rustc_nounwind]
+#[unstable(feature = "core_intrinsics", issue = "none")]
+#[cfg_attr(not(bootstrap), rustc_intrinsic)]
+#[cfg_attr(not(bootstrap), rustc_intrinsic_must_be_overridden)]
```

Comment:
Might be a bit nicer to make this a single attribute? Like, dunno, `rustc_intrinsic(must_be_overridden = true)` or so.

Reply:
I can make the `rustc_intrinsic` attribute not be required if the `rustc_intrinsic_must_be_overridden` attribute is given. I'd rather not start adding a whole attribute parser to these attributes

Reply:
By the way maybe call the attribute `#[rustc_intrinsic_no_fallback]`?

Reply:
speaking of bikeshedding, it might be nice to have a centralized `intrinsic_no_fallback` function to use as the body for these functions (a monomorophic function that just panics).